tools/accuracy_checker/accuracy_checker/config/config_reader.py

```diff
@@ -856,6 +859,11 @@ def _async_evaluation_args(launcher_entry):
         if not am.is_dir():
             launcher_entry['affinity_map'] = arguments.affinity_map
 
+    if 'device_config' in launcher_entry:
+        device_config = launcher_entry.pop('device_config')
+        if '_device_config' not in launcher_entry:
+            launcher_entry['_device_config'] = device_config
```

Comment:
it is user provided argument, underscore used for private autogenerated, so ligic should be reversed for variables

Reply:
Done